tests/test_automatic_fields.py

```diff
@@ -11,4 +11,7 @@ def name():
         return "tt_ga4_auto"
 
     def streams_to_test(self):
-        return set(self.expected_metadata().keys())
+        streams_to_test = set(self.expected_metadata().keys())
+        # We have no test data for in_app_purchases stream
+        streams_to_test.remove("in_app_purchases")
```

Comment:
Is this removed because we don't have test data?

Reply:
Yeah, talend doesn't have any app purchases in their ga4 account